.github/workflows/ai-automation.yml

```
@@ -73,13 +75,11 @@ jobs:
   issue-triage:
     if: github.event_name == 'issues'
     runs-on: ubuntu-latest
-    outputs:
-      available_labels: '${{ steps.get_labels.outputs.available_labels }}'
-      selected_labels: '${{ env.SELECTED_LABELS }}'
     steps:
+      - name: 'Get repository labels'
```

actionlint (1.7.10)

step must run script with "run" section or run action with "uses" section

(syntax-check)